.bandit.yaml

```diff
@@ -0,0 +1,7 @@
+exclude_dirs:
+  - jwst/*test*
+  - jwst/**/*test*
+  - jwst/fits_generator
+
+skips:
+  - B322
```

Comment:
This check is not needed as we are python3-only, and `input` is safe in python3.